common-util/Details/ServiceState/2StepActiveRegistration/index.jsx

```diff
@@ -55,7 +76,10 @@ const ActiveRegistration = ({
         isDisabled={btnProps.disabled}
       />
       <Text type="secondary">
-        {`Adding instances will cause a bond of ${totalBond || '--'} ETH`}
+        {`Adding instances will cause a bond of ${totalBondEthToken} ETH`}
+        {!isEthToken && (
+          <>{` and ${convertToEth((totalTokenBonds || 0).toString())} token`}</>
```

Comment:
Do we not know what the token is called? I assume not, so I suggest making it `x of token`, and link the token to the contract address on Etherscan

Reply:
On second thoughts, `x of custom token` would be even clearer @mohandast52